projects/jswap-finance/index.js

```diff
@@ -0,0 +1,45 @@
+const { fetchURL } = require('../helper/utils')
+const { getChainTvl } = require('../helper/getUniSubgraphTvl')
+const { staking } = require('../helper/staking')
+const sdk = require('@defillama/sdk')
+const BigNumber = require("bignumber.js")
+
+async function oecTvl(timestamp, ethBlock, chainBlocks) {
+    const response = await fetchURL('https://api.jswap.finance/stats/oec/tvl')
+
+    const liquidityTvl = new BigNumber(response.data.totalLiquidityUSD)
```

Comment:
What does each of these numbers represent?

Reply:
- totalLiquidityUSD, total value of liquidity
- totalTokenStakedUSD, total value of single token staked
- totalDividendStakedUSD, total value of dividend staked

Reply:
> total value of dividend staked

could you elaborate on this? what are staked dividends?

Reply:
Sure.
The platform token - `JF` will be staked into the dividends pool.
And users will get the dividend income (that from the DEX fee of platform) by proportion every day.